src/routes/docs/advanced/self-hosting/sites/+page.markdoc

```diff
@@ -0,0 +1,131 @@
+---
+layout: article
+title: Sites
+description: Harness the full power of self-hosted sites with Appwrite. Explore site deployment, management, and integration in your self-hosted environment.
+---
+
+{% partial file="configure-github-app.md" /%}
+
+# Configure Sites runtimes {% #sites %}
+
+Not all site runtimes are enabled by default. Enable the runtimes that you need and disable unused runtimes to save disk space on your server.
+To enable a runtime, add it to the `_APP_SITES_RUNTIMES` environment variable as a comma-separated list.
```

Comment:
Do we have a list of all the available runtimes?

Is there another PR somewhere for updating the list of [all environment variables](https://appwrite.io/docs/advanced/self-hosting/environment-variables) with the sites related env ars?

Reply:
I've pushed the env variables here

Unfortunately I don't have a list of available runtimes. Will check with Matej on that